fuzz/src/main/java/tech/pegasys/teku/fuzz/input/ExecutionPayloadFuzzInput.java

```diff
@@ -19,36 +19,39 @@
 import tech.pegasys.teku.infrastructure.ssz.tree.TreeNode;
 import tech.pegasys.teku.spec.Spec;
 import tech.pegasys.teku.spec.SpecVersion;
-import tech.pegasys.teku.spec.datastructures.blocks.blockbody.versions.bellatrix.BeaconBlockBodySchemaBellatrix;
-import tech.pegasys.teku.spec.datastructures.execution.versions.bellatrix.ExecutionPayloadBellatrix;
-import tech.pegasys.teku.spec.datastructures.execution.versions.bellatrix.ExecutionPayloadSchemaBellatrix;
+import tech.pegasys.teku.spec.datastructures.blocks.blockbody.versions.capella.BeaconBlockBodySchemaCapella;
+import tech.pegasys.teku.spec.datastructures.execution.versions.capella.ExecutionPayloadCapellaImpl;
+import tech.pegasys.teku.spec.datastructures.execution.versions.capella.ExecutionPayloadSchemaCapella;
 import tech.pegasys.teku.spec.datastructures.state.beaconstate.BeaconState;
 
 public class ExecutionPayloadFuzzInput
-    extends Container2<ExecutionPayloadFuzzInput, BeaconState, ExecutionPayloadBellatrix> {
+    extends Container2<ExecutionPayloadFuzzInput, BeaconState, ExecutionPayloadCapellaImpl> {
```

Comment:
ah the need to use `ExecutionPayloadCapellaImpl` here is because it's referring to the concrete implementation type. There's a bit of a battle going on with Java's type system to be honest...

`ExecutionPayloadBellatrix` is also the implementation version, just that the interface in that case is `ExecutionPayload` because it's the first version of it.  The inconsistency isn't great but I do like the simple `ExecutionPayload` name and you don't see the implementation classes in many places...